heltour/tournament/forms.py

```diff
@@ -36,6 +36,23 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         self.season = kwargs.pop('season')
+        
+        username = kwargs.pop('user')
+        
+        #try to find a SeasonPlayer object for the current season.
+        #if SeasonPlayer object already exists for the current season we know the player got accepted before 
+        #and we do not have to display the availability fields in the registration form.
+        try:
+            #the lonewolf seasons are organized in section groups. Registrations are always linked to the lonewolf open section
+            #but players might be assigned to the u1800 section. Hence we have to identify all seasons in the same section group.
+            #On the other hand, team league and 960 do not have sections and we have to check directly the season.
+            seasonsInGroup = Season.objects.filter(
+                section__in=Section.objects.filter(
+                    section_group=Section.objects.filter(season=self.season).first().section_group))
+        except:
```

Comment:
Bare excepts are dangerous. Catch only the exceptions that are necessary.

Reply:
In general, python uses snake case and I think that heltour does too. We should use the name `seasons_in_group`